packages/op-datasets/src/op_datasets/etl/intermediate/models/daily_address_summary.py

```diff
@@ -39,7 +40,7 @@
     ),
     Expression(
         alias="total_txs_success",
-        sql_expr=f"COUNT_IF({TX_SUCCESS})",
+        sql_expr=f"COUNT(IF({TX_SUCCESS}, 1, NULL))",
```

Comment:
This changed because using COUNT_IF was returning a HUGEINT type, whereas COUNT returns a BIGINT. We want the output schema to be BIGINT.